ext/app/server/lib/CommStub.ts

```diff
@@ -168,9 +171,19 @@ export class Comm  extends dispose.Disposable implements GristServerAPI, DocList
   }
 
   private async _loadInitialData(initialDataUrl: string) {
-    const content = await (await fetch(initialDataUrl)).text();
+    const response = await fetch(initialDataUrl, {
+      credentials: 'include',
```

Comment:
Ah-hah, good point. Wonder if this should be limited to `same-origin` if not configurable though? Imagine a site using grist-static to render some public csv from somewhere - they wouldn't actually want their cookies being sent along to that outside host.

Reply:
You are right, reverting it as the `same-origin` is the default one.